remix/app/lib/handler/app.handler.ts

```diff
@@ -0,0 +1,24 @@
+import { User } from '@prisma/client';
+import { Response } from '@remix-run/node';
+
+import { authenticator } from '../../config/auth.server';
+import UserRepository from '../../repositories/user.server';
+
+export default async function appHandler(
+  request: Request,
+  callback: (user: User) => Promise<unknown>
+) {
+  let userInfo: User;
+
+  const session = (await authenticator.isAuthenticated(request)) as User;
+
+  if (session?.email) {
+    userInfo = await UserRepository.findBy({ email: session.email });
+
+    return callback(userInfo);
+  } else {
+    throw new Response('You must be authenticated to access this page', {
+      status: 401,
```

Comment:
We can also use [`http-status-codes`](https://github.com/prettymuchbryce/http-status-codes) to make the status code better.

I just [learned this from Olivier days ago](https://github.com/nimblehq/js-framework-benchmark/pull/94#discussion_r1208997547) 👀

Reply:
Thanks for the suggestion I have improve in `app.handler.ts` and other file too.

Improve here:[4908b77](https://github.com/nimblehq/js-framework-benchmark/pull/112/commits/4908b779f8da278a2e906b2f62c1ed387f1f397c)